temporalio/worker/worker.py

```diff
@@ -146,6 +147,8 @@ def __init__(
                 sandboxing in order to make using a debugger easier. If false
                 but the environment variable ``TEMPORAL_DEBUG`` is truthy, this
                 will be set to true.
+            use_worker_versioning: If true, this worker opts into worker build id
+                based versioning. See the documentation at TODO for more.
```

Comment:
Unless you are gonna fill in this TODO before merge, can just remove the last sentence